src/scaffoldmaker/meshtypes/meshtype_3d_lung1.py

```diff
@@ -453,16 +451,16 @@ def generateBaseMesh(cls, region, options):
                                 lowerRightLungMeshGroup.addElement(element)
 
                 # Upper lobe elements
-                nodeIds = upperNodeIdsLeft if (lung == leftLung) else upperNodeIdsRight
+                upperNodeIds = upperLeftNodeIds if (lung == leftLung) else upperRightNodeIds
                 for e3 in range(uElementsCount3):
                     for e2 in range(uElementsCount2):
                         for e1 in range(uElementsCount1):
                             eft = eftRegular
                             nodeIdentifiers = [
-                                nodeIds[e3][e2][e1], nodeIds[e3][e2][e1 + 1], nodeIds[e3][e2 + 1][e1],
-                                nodeIds[e3][e2 + 1][e1 + 1],
-                                nodeIds[e3 + 1][e2][e1], nodeIds[e3 + 1][e2][e1 + 1], nodeIds[e3 + 1][e2 + 1][e1],
-                                nodeIds[e3 + 1][e2 + 1][e1 + 1]]
+                                upperNodeIds[e3][e2][e1], upperNodeIds[e3][e2][e1 + 1], upperNodeIds[e3][e2 + 1][e1],
+                                upperNodeIds[e3][e2 + 1][e1 + 1],
+                                upperNodeIds[e3 + 1][e2][e1], upperNodeIds[e3 + 1][e2][e1 + 1], upperNodeIds[e3 + 1][e2 + 1][e1],
+                                upperNodeIds[e3 + 1][e2 + 1][e1 + 1]]
```

Comment:
Put groups of 4 node indexes per line as for lower lobe. I don't care for pycharm's line limits within reason.
Alternatively if you want to keep within line limit, put 2 node indexes per line & be consistent in lower lobe earlier.